src/studio-home/StudioHome.test.jsx

```diff
@@ -76,6 +81,11 @@ describe('<StudioHome />', async () => {
       const { getByText } = render(<RootWrapper />);
       expect(getByText('Studio home')).toBeInTheDocument();
     });
+
+    it('should render Studio home title', () => {
+      render(<RootWrapper />);
+      screen.logTestingPlaygroundURL();
+    });
```

Comment:
Is this testcase missing a check or testing something?

Reply:
Hi @yusuf-musleh! Thank you for the review.
Cleaned here: 0d58997747fc7830424748e6edb0edd34de52b34